.circleci/config.yml

```diff
@@ -54,8 +54,16 @@ jobs:
       - db-setup
       - ruby/rubocop-check
       - run:
-          # Ensure sample_data runs without issues
-          name: Seed with sample data
+          # Ensure db:migrate runs without issues
+          name: Migrate database
+          command: bin/rails db:migrate
```

Comment:
This might be a little controversial? I think a common opinion is that migrations are ephemeral, in the sense that they serve a purpose in the moment a feature is delivered, but there is no need to keep testing that all migrations work from the beginning of time (i.e. migrating an empty database).

I personally think it is important to maintain migrations because you may need to run them on an empty database sometimes. For example, consider a project that has a single acceptance environment for a couple months, then sets up a production environment. When you deploy to heroku in that new environment, the heroku release task is going to run the entire history of migrations from the beginning on that blank database.

But, if we don't agree there is enough value there, I can remove this. Maintaining historical migrations is not a burden that all teams will want to carry (and for certain types of migrations, might not be possible).